### DISCUSSION TITLE
有安全公司吗？币被转走了，求线索

### DISCUSSION THREAD

**Discussion by star1983** [filecoin-project/community-china#98] [应用开发]
服务器上的FIL被提现至陌生地址上去了，服务器上有私钥，本地也有一份，这个有什么办法能查到是通过什么方式转出？IP地址之类的吗？

**Comment by phantom-rabbit** [comment]
没有办法查询IP，应为是分布式的，大家都通过网络转发这条消息。可以根据消息类型知道是什么方式转出。至于你想追踪是谁转走了你的币，那就需要对这个接收地址一直监控。只要他进入中心化交易所市场就可以知道是谁转走了(一般中心化交易所需要实名认证)。

**Reply by KaixuanZhao** [reply]
RRMine，雅典娜矿池，全网算力领先1050PB，需要B端C端挖矿方案的滴滴我，欢迎加入社群Wechet：kkmmjjnn00

**Comment by yhhwpp** [comment]
很同情你

**Comment by maxvint** [comment]
哎，这个只能同情了，希望官方的钱包安全机制，尽快完善起来吧，至少加个密码也行呀。

**Comment by Chenhongwe** [comment]
独家签名机方案，保护你的私钥安全，另单卡单U单台封装效率7T，软硬一体方案，详询v15088695255
